### PR TITLE
Feature/style 9 npmize

### PR DESCRIPTION
**Pull request by hrag**

### CODE REVIEW

File: lib/scss/_cards.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     transition: 0.3s all ease-in-out;
 
-    @include media($desktop) {
+    @include viewport($sd-desktop) {
```
Comment (blopker) [inline lib/scss/_cards.scss]: This will probably need to be prefixed too.

Comment (blopker) [reply lib/scss/_cards.scss]: "this" == `viewport`

File: lib/scss/dependencies/_legacy_dependencies.scss [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+// @mixin ttam-placeholder {
+//   $placeholders: ":-webkit-input" ":-moz" "-moz" "-ms-input";
+//   @each $placeholder in $placeholders {
+//     &:#{$placeholder}-placeholder {
+//       @content;
+//     }
+//   }
+// }
+
+@function assign-inputs($inputs, $pseudo: null) {
```
Comment (blopker) [inline lib/scss/dependencies/_legacy_dependencies.scss]: functions will need prefixes.

File: src/lib/loading.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
+function escape(unsafeText) {
+    const safeText = $('<div />').text(unsafeText).text();
+    return safeText;
+}
+
+TTAM.loading = {
```
Comment (blopker) [inline src/lib/loading.js]: This is going to just be exported in another PR, right?

File: lib/scss/dependencies/_legacy_dependencies.scss (hunk 2 above)
Comment (hrag) [reply lib/scss/dependencies/_legacy_dependencies.scss]: Roger that!

File: src/lib/loading.js (hunk 3 above)
Comment (hrag) [reply src/lib/loading.js]: Sorry for the lag on this - yeah, I'm going to add these changes in STYLE-7.
